src/main/python/rlbot/utils/rendering/rendering_manager.py

```diff
@@ -211,7 +212,29 @@ def get_rendering_manager(self, bot_index=0):
     def __wrap_float(self, number):
         return Float.CreateFloat(self.builder, number)
 
-    def __create_vector(self, x, y, z=None):
-        if z is None:
-            z = 0
+    """Supports Flatbuffers Vector3, cTypes Vector3, list/tuple of numbers, or passing x,y,z (z optional)"""
+    def __create_vector(self, *vec):
+        if (isinstance(vec[0], int) or isinstance(vec[0], float)) and (isinstance(vec[1], int) or isinstance(vec[1], float)):
```

Comment:
you should a length check.

instead of all these instance checks for int and float (also you can check for number)
do a length check for 2 or 3
then later do the check for 3 for z

then you can do the other type checks